ocp_resources/data_source.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         teardown=True,
         yaml_file=None,
         delete_timeout=TIMEOUT_4MINUTES,
+        label=None,
```

Comment:
either you add it to all resources or do not add it as we already have `kwargs` which are passed.